compiler/resolution/ResolutionCandidate.cpp

```diff
@@ -1064,6 +1095,15 @@ void explainCandidateRejection(CallInfo& info, FnSymbol* fn) {
     case RESOLUTION_CANDIDATE_IMPLICIT_WHERE_FAILED:
       USR_PRINT(fn, "because an argument was incompatible");
       break;
+    case RESOLUTION_CANDIDATE_INTERFACE_CONSTRAINTS_NOT_SATISFIED:
+      USR_PRINT(fn, "because interface constraint(s) were not satisfied\n");
```

Comment:
Isn't it really that the constrained generic *function*'s constraints were not satisfied? I would expect the error you have here for `int implements Addable` when `int` doesn't have an `add` function, e.g.

In any case I think we would like to show more information about what constraint was not satisfied.